### ISSUE TITLE
Incorrect website documentation on timetables.txt formatting

### ISSUE THREAD

**Issue by jack-quartic** [BlinkTagInc/gtfs-to-html#167]
Hello! I wrote a script for a transit agency client to build them a timetables.txt file, so they could run it through your tool and stack routes. After following the schema for timetables.txt and running the tool, it wouldn't generate any timetables, or if I set _allowEmptyTimetables: true_ they would just be blank.

After some troubleshooting, I noticed the timetables.txt documentation indicates that _start_date_ and _end_date_ need to be in the format _YYYY-MM-DD_, however, the examples at the bottom of the page show YYYYMMDD. After switching to using YYYYMMDD, I can generate timetables w/o issue. I just wanted to make you aware of the documentation discrepancy.

Really neat tool, thank you!

- Jack C.
- 
![image](https://github.com/user-attachments/assets/d6af2242-1844-4b76-bad1-183ef7189b68)
![image](https://github.com/user-attachments/assets/6a06e9e6-0273-49a9-b8f1-44aa6800a64d)

**Comment by brendannee** [comment]
Thanks so much for pointing out this issue - you are probably not the only one tripped up by this documentation.

I published and updated version https://github.com/BlinkTagInc/gtfs-to-html/releases/tag/2.7.0 and updated documentation to resolve this.

* The documentation now specified YYYYMMDD as the format needed.
* However, the library is updated to accept either YYYY-MM-DD or YYYYMMDD. YYYYMMDD is the preferred and correct format, but either should work.
* The library is updated to throw an error when an incorrect date format is used with details on the file and line number.

Try it out and let me know how it works.

I'd be interested to see the script you wrote for creating timetables.txt and ideas on how to make this process easier.

**Comment by jack-quartic** [comment]
Glad to help. 🙌 Amazing, thank you for the all the updates, especially the library ones. 

Once I've fleshed out the rest of the script (add timetable_label for stacked routes, and clean it up a bit) I'd be glad to share.

I'll test your updates and let you know.

**Comment by jack-quartic** [comment]
@brendannee Works great. Runs fine regardless of hyphen inclusion. I was trying to force the error check you mentioned adding in the 3rd bullet, and when I ran it using YYYYDDMM instead of MMDD, I just get the following warning, which is the same error I got before I submitted this issue:

`Warning: No trips found for route_id=71-180, direction_id=0, service_ids=[], timetable_id=118`

I don't see any error with details on file and line no. However, I am running this tool as a python subprocess, and spitting out stdout and stderr to the VS Code terminal. Not sure if this may be why I don't see the error.
